src/main/docs/guide/introduction/upgrading.adoc

```diff
@@ -117,6 +117,75 @@ To use the new module, you need to add the following dependency:
 
 dependency:io.micronaut.kubernetes:micronaut-kubernetes-discovery-client[]
 
+=== Transaction Management
+
+For Micronaut 2.0, we recommend you to use the native Micronaut-based transaction management instead of other alternatives such as Spring Transaction Mangement.
+
+You will use https://docs.oracle.com/javaee/7/api/javax/transaction/Transactional.html[Java EE 7 javax.transaction.Transactional annotation] and https://micronaut-projects.github.io/micronaut-data/latest/api/index.html[io.micronaut.transaction.annotation.ReadOnly] to mark your transaction demarcations.
```

Comment:
Please use the `javaee` variable for this link rather than hard coding the URL (see other links to for examples)

Reply:
done